sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -179,6 +188,28 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
 
         .. versionadded:: 0.22
 
+    kind : {"average", "individual", "both"}, default="average"
+        Whether to plot the partial dependence averaged across all the samples
+        in the dataset or one line per sample or both individual lines and the
+        average dependence at the same time.
+
+        - kind="average" results in the traditional PD plot;
+        - kind="individual" results in the ICE plot.
+
+       Note that the fast method="recursion" option is only available for
+       kind="average". Plotting individual dependencies requires using the
+       slower method="brute" option.
+
+        .. versionadded:: 0.23
+
+    subsample : float, int or None, default=1000
+        Sampling for ICE curves when `kind` is 'individual' or 'both'.
+        If `float`, should be between 0.0 and 1.0 and represent the proportion
+        of the dataset to be used to plot ICE curves. If `int`, represents the
```

Comment:
Perhaps note that the full dataset is still used to calculate partial dependence...?

Reply:
Thanks for the suggestion. Done in c44f12eb8d0cbe15a65b0bb9764fca63b5518af9